Sources/Plasma/PubUtilLib/plAgeDescription/plAgeDescription.cpp

```diff
@@ -491,11 +491,8 @@ void    plAgeDescription::CopyFrom(const plAgeDescription& other)
 
 bool plAgeDescription::FindLocation(const plLocation& loc) const
 {
-    for (const plAgePage& page : fPages)
+    return std::any_of(fPages.begin(), fPages.end(), [this, loc](const plAgePage& page)
     {
-        plLocation pageLoc = CalcPageLocation(page.GetName());
-        if (pageLoc == loc)
-            return true;
-    }
-    return false;
+	    return loc == CalcPageLocation(page.GetName());
```

Comment:
```suggestion
	    return CalcPageLocation(page.GetName()) == loc;
```

Let's not invert the order of the comparison.